tools/buildHeaders/bin/makeExtinstHeaders

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
I've gotten some push back on bash being platform dependent.  Is there a reason to use bash instead of python?

Reply:
I think I was following the existing script example.  I can rewrite in Python3.
(and if we go that route, we can eliminate the need for dos2unix)

Reply:
Rewritten as Python3